_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -25,8 +25,8 @@ endif::[]
 
 == Intro
 
-This guide will walk you through the process of configuring a production-grade AWS account structure, including how to
-manage multiple environments, users, permissions, audit logging, and more.
+This guide will walk you through the process of configuring a production-grade AWS account structure using the Gruntwork
+Landing Zone solution, including how to manage multiple environments, users, permissions, audit logging, and more.
```

Comment:
We only talk about Gruntwork's Landing Zone solution in the deployment walkthrough, so for now, introduce the concept more generically:

```suggestion
This guide will walk you through the process of configuring a production-grade AWS account structure, including how to manage multiple environments, users, permissions, and audit logging. We'll also discuss how to implement a _Landing Zone_ solution that lets you quickly spin up new AWS accounts that all implement a _security baseline_ that enforces your company's policies. 
```